rxpager/src/test/java/com/schibstedspain/android/rxpager/PagerTest.java

```diff
@@ -34,131 +36,125 @@
 
   private Pager<TokenPage<String>, String> pager;
   @Mock
-  private Func1<String, Observable<TokenPage<String>>> getPageMock;
+  private Function<String, Observable<TokenPage<String>>> getPageMock;
 
   @Before
   public void setUp() {
-    pager = new Pager<>(null, (s, stringTokenPage) -> stringTokenPage.getNextPageToken(), getPageMock);
+    pager = new Pager<>("0", (s, stringTokenPage) -> stringTokenPage.getNextPageToken(), getPageMock);
   }
 
   @Test
-  public void getPageObservableShouldReturnJustOnePageIfThereIsOnlyOne() {
+  public void getPageObservableShouldReturnJustOnePageIfThereIsOnlyOne() throws Exception {
     givenThereIsOnePage();
 
-    TokenPage<String> firstPage = pager.getPageObservable()
-        .toBlocking()
-        .single();
+    TokenPage<String> firstPage = pager.getPageObservable().blockingFirst();
```

Comment:
Use `blockingSingle` instead please, first would allow the observable to emit more than once.

Reply:
done